src/views/Items/Item.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+import customPropTypes from '../../utils/customPropTypes';
+
+export default class Item extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    description: customPropTypes.mutuallyExclusive(['children']),
+    extra: PropTypes.node,
+    heading: PropTypes.node.isRequired,
+    image: PropTypes.node,
+    meta: PropTypes.node,
+  };
+
+  render() {
+    let heading = <div className='header'>{this.props.heading}</div>;
+    let meta = <div className='meta'>{this.props.meta}</div>;
+    let description = <div className='description'>{this.props.children || this.props.description}</div>;
+    let extra = <div className='extra'>{this.props.extra}</div>;
+    let content = (
+      <div className='middle aligned content'>
+        {this.props.heading && heading}
+        {this.props.meta && meta}
+        {this.props.children && description || this.props.description && description}
+        {this.props.extra && extra}
+      </div>
+    );
+    let hasContent = !!this.props.heading || !!this.props.meta || !!this.props.extra || !!this.props.children
+    || !!this.props.description;
+
+    let classes = classNames(
+      'sd-item',
+      this.props.className,
+      'item',
+    );
+    return (
+      <div {...this.props} className={classes}>
```

Comment:
Component props used for Stardust purposes should be removed from `this.props` before spreading.  Only the user's props should be spread.  Since this component takes in:

```
children
className
description
extra
heading
image
meta
```

We should remove those props before spreading them onto the `Item`.  Otherwise, we'll be spreading components and other unintended things:

```jsx
let props = _.clone(this.props);
delete props.children;
delete props.className;
delete props.description;
delete props.extra;
delete props.heading;
delete props.image;
delete props.meta;

// then

<div {...props} ...
```

Thinking about this, I believe we should always remove `static PropTypes` props, except for `static defaultProps` before spreading.  This way only "extra" user defined props are spread.  Linking #69 spreadable props util here for this reference.

For now, add the clone/delete step before spreading.